setup.py

```diff
@@ -3,10 +3,7 @@
 import re
 import sys
 import warnings
-try:
-    from setuptools import setup
-except:
-    from distutils.core import setup
+from setuptools import setup, find_packages
```

Comment:
Wasn't the try/except way a more forgiving way of doing things? Or do you have requirements for tonic and if they're met then the new `from setuptools import setup, find_packages` should work no matter what?